### PR TITLE
Add helpful plugins and modified README file.

### PR DESCRIPTION
**Pull request by Yidnekachew-cmd**

### CODE REVIEW

**Review by Yidnekachew-cmd — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Highlights
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

Required Changes ♻️
Check the comments under the review.

 Please from the project requirements, you should use the linters provided by microverse to check your project and run checks on your pull request. But as you can see you have some errors in your project.


Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      ><i class="fas fa-at"></i> Send a mail</a
+    >
+    <a href="tel:555-555-5555" class="btn contact-details"
+      ><i class="fas fa-mobile-alt"></i> Call me</a
+    >
+  </div>
+</section>
+
+
+
```
Comment (Yidnekachew-cmd) [inline index.html:192]: You don't need extra space between every tag. Make sure you clean your code once before pushing it to the remote repository. kindly delete these free spaces.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    font-size: 2.2rem;
+    padding: 2rem;
+  }
+  
+  .nav-list a:hover {
+    background: var(--main-blue);
+  }
+  
+
+
```
Comment (Yidnekachew-cmd) [inline style.css:128]: Here also there is unnecessary free space. Make sure you delete this free space and clean your CSS.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+      justify-content: center;
+    }
+  
+    .nav-list {
+      margin: 0 1rem;
+    }
+  }
+  
+  .nav-list a {
+    display: block;
```
Comment (Yidnekachew-cmd) [inline style.css:118]: You only need two spaces before the block code.

**Review by andgarzonmal — APPROVED** [review]
Hi @Yidnekachew-cmd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**